spread.yaml

```diff
@@ -848,14 +848,15 @@ suites:
             distro_update_package_db
             distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils
             snap install ubuntu-image --classic
-
-            # TODO: replace with distro package once 2.45 is in stable
-            dpkg -i "$SPREAD_PATH"/../snapd_*.deb
-
+        prepare-each: |
             #shellcheck source=tests/lib/nested.sh
             . "$TESTSLIB/nested.sh"
             nested_prepare_env
-
+        restore-each: |
```

Comment:
hmm does restore-each get run _before_ the debug shell or after? I think it's after in which case this change is fine but I don't recall directly and the docs on spread don't clarify when exactly debug shells get started and I'm too lazy to go look at the code :smile:

Reply:
Correct, the restore happens after the debug.